app/controllers/application_controller/performance.rb

```diff
@@ -203,6 +203,7 @@ def perf_set_or_fix_dates(options)
     edate_daily = edate.hour < 23 ? edate - 1.day : edate
     options[:edate_daily] = edate_daily
 
+    options[:typ] = "Hourly" if options[:typ] == "Daily" && edate_daily < sdate_daily
```

Comment:
I think that looking at the PR title, having this fix here might be misleading. so either the PR title should be adjusted, or this should move to a separate fix.